server/src/database/index.ts

```diff
@@ -0,0 +1,15 @@
+import sequelize from './config/connection';
+import {
+  User, Order, Product, Category,
```

Comment:
import the `ProductOrder` model to make sure it's synced when we call the sync script